samples/drivers/dac/boards/gd32f450i_eval.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2021 BrainCo Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	zephyr,user {
+		dac = <&dac>;
+		dac-channel-id = <0>;
+		dac-resolution = <12>;
+	};
+};
+
+&dac {
+	status = "okay";
+};
```

Comment:
this should be done at board level (board will not `CONFIG_DAC=y`, so driver won't be enabled)

Reply:
done.